src/main/java/core/basesyntax/service/impl/ShopFileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ShopFileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class ShopFileReaderImpl implements ShopFileReader {
+    @Override
+    public String read(String path) {
+        StringBuilder builder = new StringBuilder();
+        String line = "";
+        try (java.io.FileReader fr = new java.io.FileReader(path)) {
```

Comment:
It's better to use only 'FileReader' instead of 'java.io.FileReader'. Make sure to import 'java.io.FileReader' at the beginning of the file.